### PR TITLE
Refactor with hooks

### PR DESCRIPTION
**Pull request by studiosnabeel**
### Tasks Completed

- Refactor Calculator class component to function component with hooks.
- Style the buttons with hover effects.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,123 +1,101 @@
 /* eslint-disable react/no-unused-state */
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prefer-stateless-function */
```
Comment (FrederickMih) [inline src/components/Calculator.js]: - [ ] It's a bad practice to disable Linters checks locally while coding. Linters detect bugs :bug:  in our codebase and flag the error. There are great tools. Kindly fix the errors.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @studiosnabeel ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

- App still works after refactoring :+1: 
- Good commit history :heavy_check_mark: 



## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions

- Please kindly double-check the entire README file and make sure that all the information is in line with this project. You just copy/paste without customizing the entire file.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (studiosnabeel) [reply src/components/Calculator.js]: @FrederickMih Thank you for pointing that out. I forgot to remove these after refactoring. I have removed them now.

**Review by Reem-lab — APPROVED** [review]
# Hi @studiosnabeel, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- Followed Gitflow ✅
- Great working on the functionality 🔥🚀🎉
- Professional README file 💯✨🔥
- Descriptive PR ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**